common/app/views/fragments/nav/userAccountLinks.scala.html

```diff
@@ -14,49 +13,47 @@
     </li>
 
     <li class="menu-item u-h js-navigation-account-details">
-        <details>
-            <summary class="menu-item__title"
-                     data-link-name="nav2 : my account">
-                <i class="menu-item__toggle"></i>
-                my account
-            </summary>
+        <button class="menu-item__title js-navigation-toggle"
```

Comment:
Then we will have two links to the same place near eachother. Do you think thats a problem? 

I am not sure if it should link to anything right now. This really is just a label.